deno_webgpu/query_set.rs

```diff
@@ -43,9 +43,10 @@ impl GPUQuerySet {
 
     #[fast]
     fn destroy(&self) -> Result<(), JsErrorBox> {
-        Err(JsErrorBox::generic(
-            "This operation is currently not supported",
-        ))
+        // TODO(#6495): Destroy the query set. Until that is supported, it is
```

Comment:
thought: We normally use full GH issue links here, but I'm not sure that this is actually a problem. CC @gfx-rs/wgpu.